security/pfSense-pkg-snort/files/usr/local/www/snort/snort_rulesets.php

```diff
@@ -44,29 +39,31 @@
         exit;
 }
 
-if (isset($id) && $a_nat[$id]) {
-	$pconfig['enable'] = $a_nat[$id]['enable'];
-	$pconfig['interface'] = $a_nat[$id]['interface'];
-	$pconfig['rulesets'] = $a_nat[$id]['rulesets'];
-	if (empty($a_nat[$id]['autoflowbitrules']))
+$a_nat = config_get_path("installedpackages/snortglobal/rule/{$id}", []);;
```

Comment:
Looks like a bonus semicolon here that can be removed.

Reply:
Copy and paste caused it. Removed the extra one.